ArduPlane/Log.cpp

```diff
@@ -102,13 +102,19 @@ void Plane::Log_Write_Control_Tuning()
         synthetic_airspeed = logger.quiet_nan();
     }
 
+    int16_t pitch = ahrs.pitch_sensor - g.pitch_trim * 100.0f;
+#if HAL_QUADPLANE_ENABLED
+    if (quadplane.in_vtol_mode()) {
+    pitch = ahrs.pitch_sensor;
+    }
+#endif      
```

Comment:
I don't think you need this case, the existing code is already wrong for quadplanes.

Reply:
I don't understand the reasoning, both this and the un-offset value are not correct for VTOL. It's just going to be confusing.

Reply:
looks good to me in the logs now